portafly/src/components/pages/accounts/ExportAccountsButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { useTranslation } from 'i18n/useTranslation'
+import { ExportIcon } from '@patternfly/react-icons'
+import { CSVLink } from 'react-csv'
+import { IDeveloperAccount } from 'types'
+import { Button } from '@patternfly/react-core'
+
+interface Props {
+  data?: IDeveloperAccount[]
+}
+const ExportAccountsButton: React.FunctionComponent<Props> = ({ data }) => {
+  const { t } = useTranslation('accountsIndex')
+
+  const exportHeaders = [
+    { label: 'ID', key: 'id' },
+    { label: t('accounts_table.admin_header'), key: 'admin_name' },
+    { label: t('accounts_table.group_header'), key: 'org_name' },
+    { label: t('accounts_table.state_header'), key: 'state' },
+    { label: t('accounts_table.applications_header'), key: 'apps_count' },
+    { label: t('accounts_table.created_header'), key: 'created_at' },
+    { label: t('accounts_table.updated_header'), key: 'updated_at' }
+  ]
+
+  return (
+    data ? (
+      <CSVLink
+        filename="accounts.csv"
+        headers={exportHeaders}
+        data={data}
+        className="pf-c-button pf-m-link"
+        aria-label={t('export_accounts_button_aria_label')}
+      >
+        <span className="pf-c-button__icon pf-m-start"><ExportIcon /></span>
+        {t('export_accounts_button')}
+      </CSVLink>
```

Comment:
Instead of rendering the different button I think we could [override the onClick](https://github.com/react-csv/react-csv#--onclick-props) and add a disabled style:
```suggestion
// Button should be disabled also when there are no accounts (empty)
// const isDisabled = data && data.length
      <CSVLink
        filename="accounts.csv"
        headers={exportHeaders}
        data={data}
        className={`pf-c-button pf-m-link ${isDisabled ? 'pf-m-disabled'} : ''`}
        aria-label={t('export_accounts_button_aria_label')}
        onClick={() => !isDisabled}
      >
        <span className="pf-c-button__icon pf-m-start"><ExportIcon /></span>
        {t('export_accounts_button')}
      </CSVLink>
```